TYPO3.TYPO3CR/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -75,6 +75,10 @@ additionalProperties:
     'options':
       type: dictionary
 
+    'fusion':
```

Comment:
I agree to your point but TBH I don't think it's a better solution to "hide" this defacto dependency within some option that is not part of the schema.
IMO a good solution would be: extensible schemas. That would allow the Neos or Fusion package to extend the schema.
It shouldn't be to hard to adjust the `SchemaValidator` accordingly. In fact, it might already be able to do that.